Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -27,6 +27,9 @@ public enum ModuleError: Swift.Error {
     /// One or more referenced targets could not be found.
     case modulesNotFound([String])
 
+    /// Invalid custom path
+    case invalidCustomPath((target: String, path: String))
```

Comment:
Two enum associated values feel more natural than an anonymous tuple
```suggestion
    case invalidCustomPath(target: String, path: String)
```